text/0060-refashion-updates-apis.md

```diff
@@ -112,7 +112,70 @@ The motivation is to stabilize the current `update` resource to a version that c
 >
 > 💡 `Settings` is updated to be more precise with the name `settingsUpdate`.
 
-#### 4. Examples
+#### 4. `details` field object
+
+##### documentsAddition
+
+| name     | description |
+| -------- | --------    |
+| receivedDocuments | Number of documents received. |
+| indexedDocuments  | Number of documents finally indexed. |
+
+
+##### documentsPartial
+
+| name     | description |
+| -------- | --------    |
+| receivedDocuments | Number of documents received. |
+| indexedDocuments  | Number of documents finally indexed. |
+
+
+##### documentsDeleted
+
+| name     | description |
+| -------- | --------    |
+| receivedDocuments | Number of documents received.  |
+| deletedDocuments | Number of documents finally deleted. |
+
+##### indexCreation
+
+| name     | description |
+| -------- | --------    |
+| primaryKey  | Value for the `primaryKey` field into the POST index payload.|
+
+
+##### indexUpdate
+
+| name     | description |
+| -------- | --------    |
+| primaryKey | Value for the `primaryKey` field into the PUT index payload. |
+
+##### indexDeletion
+
+| name     | description |
+| -------- | --------    |
+| deletedDocuments    | Number of deleted documents. Should be all documents contained in the deleted index. |
```

Comment:
Is this one really necessary? We don't interact with the index when deleting it, we simply remove it from fs, so this information require that we open and read from the index before we delete it. It's possible to do it, just making sure that this is necessary.

Reply:
It seems interesting to me to give this information to have consistency on the information returned when documents are deleted. Do you see any potential problems or technical constraints to open and read the index before cleaning it of these documents?

Reply:
not really